e2e/helpers/connection-actions.ts

```diff
@@ -0,0 +1,162 @@
+import { ElementHandle } from 'puppeteer';
```

Comment:
Please, rename this file to `relationship-actions.ts`, and all occurrences of the word `connection` to `relationship` .  We are deprecating the term `connections`.  And while it still exists in many places (most importantly the DB), let's try to introduce the newly "accepted" term to new development.

Reply:
Done.